snudda/init/init.py

```diff
@@ -1346,30 +1346,32 @@ def define_striatum(self,
         # ================================================================
```

Comment:
No reason to keep these comments. I suggest to remove the if statement and the comments

Reply:
also the conductance is set to ChINgGABA while the type is set to ACh - what will this do?

Reply:
Removed

Reply:
Changed to ChINgACh